js/Events/logInEvents.js

```diff
@@ -1,44 +1,47 @@
 const addEventsLogIn = function (){
-
     const submitFormEvent = function (event) {
     
-    event.stopPropagation();
-   event.preventDefault();
+        event.stopPropagation();
+        event.preventDefault();
 
-    const userNameInput = document.querySelector('#userNameInput');
-    const passwordInput = document.querySelector('#passwordInput');
+        const userNameInput = document.getElementById('userNameInput');
+        const passwordInput = document.getElementById('passwordInput');
+        const userNameValue = userNameInput.value;
+        const passwordValue = passwordInput.value;
 
-    const userNameValue = userNameInput.value;
-    const passwordValue = passwordInput.value;
+        if(checkCredentials(userNameValue, passwordValue)){
+            const app = document.getElementById('app');
+            app.innerHTML = EvaluationsPage();
+            addEventsEvaluations();
+        } else {
+            displayAlert(event.target, "Wrong username or password!");
+        }
 
-    if(checkCredentials(userNameValue, passwordValue)){
-        event.target.submit();
-    } else {
-        displayAlert(event.target, "Wrong username or password!");
-    }
-    
-        
-};
+    };
 
-const displayAlert = function(parent, message) {
-    var alertLabel = document.createElement('p');
-    alertLabel.className = "alert";
-    alertLabel.innerHTML = message;
-    parent.appendChild(alertLabel);
-};
+    const displayAlert = function(parent, message) {
+        var alert = document.querySelector('.alert');
 
-const checkCredentials = function(username, password){
-    
-    if(username ==="adrian" && password ==="frunza"){
-        return true;
-    } else {
-        return false;
-    }
-};
+        if(!alert){
+            const alertLabel = document.createElement('p');
+            alertLabel.className = "alert";
+            alertLabel.innerHTML = message;
+            parent.appendChild(alertLabel);
+        }  
 
-const logInForm = document.querySelector('#logInForm');
+    };
 
-logInForm.addEventListener('submit',submitFormEvent);
-};
+    const checkCredentials = function(username, password){
+        var logInObj = getLogInData();
+        if(username === logInObj.username && password === logInObj.password){
```

Comment:
can you think of any way to make this entire if block a one liner?

Reply:
return (username === logInObj.username && password === logInObj.password) ? true : false ;

Reply:
even shorter, but you are getting close

Reply:
return username === logInObj.username && password === logInObj.password ;

Reply:
there you go :)